Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.21-alpine AS builder
+FROM golang:1.23-alpine AS builder
 ARG COSMO_VERSION=3.9.2
```

Comment:
Oh, I never ended up using this. This line is unnecessary.

Reply:
Removed this line.